### ISSUE TITLE
ENUM statements not parsed correctly

### ISSUE THREAD

**Issue by gustason** [BalticAmadeus/AblFormatter#201]
For example,
``` abl
ENUM Weather:
  define ENUM Rainy
              Default = Sunny
              Cloudy
              Snowy
              Apocalyptic.
END ENUM.		
```
![image](https://github.com/user-attachments/assets/ea61b949-c978-4a25-b59e-2048fbe1ac03)

ENUM flags do not work either.
For example,
``` abl
ENUM LiteraryMood FLAGS:
  define ENUM None = 0
              Joyful
              Melancholy
              Heroic
              Whimsical
              Epic
              JoyfulMelancholy = Joyful,Melancholy
              Tragic = Epic.
END ENUM.
```
![image](https://github.com/user-attachments/assets/21de48ae-357f-4d18-ba1a-8942f0968358)

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/80

**Comment by eglekaz** [comment]
Fixed with version 0.0.40 of tree-sitter-abl
